src/cljs/athens/electron/dialogs.cljs

```diff
@@ -0,0 +1,96 @@
+(ns athens.electron.dialogs
+  (:require
+    [athens.athens-datoms :as athens-datoms]
+    [athens.db :as db]
+    [athens.electron.utils :as utils]
+    [datascript.core :as d]
+    [datascript.transit :as dt]
+    [re-frame.core :as rf]))
+
+
+(def electron (js/require "electron"))
+(def remote (.. electron -remote))
+(def fs (js/require "fs"))
+(def dialog (.. remote -dialog))
+(def path (js/require "path"))
+
+
+
+(rf/reg-event-fx
+  :fs/open-dialog
+  (fn [{:keys [db]} _]
+    (js/alert (str "No DB found at " (:db/filepath db) "."
+                   "\nPlease open or create a new db."))
+    {:dispatch-n [[:modal/toggle]]}))
+
+
+(defn move-dialog!
+  "If new-dir/athens already exists, no-op and alert user.
+  Else copy db to new db location. When there is an images folder, copy /images folder and all images.
+    file:// image urls in block/string don't get updated, so if original images are deleted, links will be broken."
+  []
+  (let [res     (.showOpenDialogSync dialog (clj->js {:properties ["openDirectory"]}))
+        new-dir (first res)]
+    (when new-dir
+      (let [curr-db-filepath @(rf/subscribe [:db/filepath])
+            base-dir         (.dirname path curr-db-filepath)
+            base-dir-name    (.basename path base-dir)
+            curr-dir-images  (.resolve path base-dir utils/IMAGES-DIR-NAME)
+            new-dir          (.resolve path new-dir base-dir-name)
+            new-dir-images   (.resolve path new-dir utils/IMAGES-DIR-NAME)
+            new-db-filepath  (.resolve path new-dir utils/DB-INDEX)]
```

Comment:
I think a lot of this code can be refactored and reused in other functions such as in `:boot` or `create-dialog!`, especially around file and directory creation.